modules/core/shared/src/main/scala/Session.scala

```diff
@@ -300,6 +300,10 @@ trait Session[F[_]] {
    */
   def parseCache: Parse.Cache[F]
 
+  /**
+   * Closes any prepared statements evicted as part of this session.  Statements are evicted when the cache limit is reached.
+   */
+  private[skunk] def clearEvicted: F[Unit]
```

Comment:
I don't love this but I didn't know where else to put it and this solution seemed the most fool-proof implementation with the smallest change to the API.

Reply:
Yes, fixing.